aws-analytics-pinpoint/src/test/java/com/amplifyframework/analytics/pinpoint/AnalyticsClientTest.kt

```diff
@@ -125,4 +129,27 @@ class AnalyticsClientTest {
         analyticsClient.recordEvent(pinpointEvent)
         coVerify(exactly = 1) { eventRecorder.recordEvent(pinpointEvent) }
     }
+
+    @Test
+    fun `test submitEvents`() = runTest {
+        coEvery { eventRecorder.submitEvents() } returns listOf()
+        analyticsClient.flushEvents()
+        coVerify(exactly = 1) { eventRecorder.submitEvents() }
+    }
+
+    @Test
+    fun `test addGlobalAttributes`() {
+        val attributeName = "attributeName"
+        val attributeValue = "attributeValue"
+        analyticsClient.addGlobalAttribute(attributeName, attributeValue)
+        assertEquals(analyticsClient.getGlobalAttributes()[attributeName], "attributeValue")
+    }
+
+    @Test
+    fun `test addGlobalMetrics`() {
+        val metricName = "attributeName"
+        val metricValue = 1.0
+        analyticsClient.addGlobalMetric(metricName, metricValue)
+        assertEquals(analyticsClient.getGlobalMetrics()[metricName], 1.0)
```

Comment:
nit expected value first then actual and using variable in place of hardcoded value:
```suggestion
        assertEquals(metricValue, analyticsClient.getGlobalMetrics()[metricName])
```